src/components/molecules/NavSearchBar/NavSearchBar.scss

```
@@ -18,58 +29,66 @@
   }
 }
 
-.nav__search input {
-  width: 18.125rem;
-  text-align: left;
-  background-color: $saturated-black;
-  border: solid 2px rgba(0, 0, 0, 0.13);
-  font-size: 0.9em;
-}
-
-.nav-search-results {
-  z-index: z(navbar-nav-search-results);
-
+.nav-dropdown {
   position: fixed;
-  top: 60px;
-
-  min-width: 18.125rem;
-
+  z-index: 1;
+  top: 0;
   height: auto;
-  max-height: 75vh;
-  overflow-y: auto;
-
-  padding: 20px;
-  box-shadow: 0 10px 30px 0 rgba(0, 0, 0, 0.5);
-  border-radius: 0 0 24px 24px;
-  transition: transform 400ms cubic-bezier(0.23, 1, 0.32, 1);
+  width: $search-results-width;
+  margin-top: 66px;
```

What does this margin relate to? I expect it's the height of the `NavBar`? If so, we should use  `$navbar-height`

```suggestion
  margin-top: $navbar-height;
```

---

fixed: https://github.com/sparkletown/sparkle/pull/1328/commits/e21e67445665543be0fab0f660743356141c30c7